### PR TITLE
Localize status sensor

### PR DESCRIPTION
**Pull request by astrandb**
Sensor status can now be localized so "CHARGING" is "Charging" in english and "Laddar" in swedish. Bokmål remains to be fixed.

### CODE REVIEW

**Comment by tmjo** [comment]
Looks good. We should add the UNKNOWN which is actually ERROR, see my [PR ](https://github.com/fondberg/pyeasee/pull/26)in pyeasee lib  I can fix the nb file.

**Comment by tmjo** [comment]
If you don't mind, please just put in the nb as well. If not I can put it in later.

```
{
  "state": {
    "easee_status": {
      "STANDBY": "Ventemodus",
      "PAUSED": "Pause",
      "READY_TO_CHARGE": "Klar til lading",
      "CAR_CONNECTED": "Bil tilkoblet",
      "CHARGING": "Lader"
    }
  }
}
```

**Review by tmjo — APPROVED** [review]

**Comment by fondberg** [comment]
Will this mean automations no longer can use the english capitalised statuses?

File: custom_components/easee/entity.py [hunk 1]
```diff
@@ -190,6 +192,11 @@ def get_value_from_key(self, key):
 
     async def async_update(self):
         """Get the latest data and update the state."""
+#         self._translations = await async_get_translations(
```
Comment (fondberg) [inline custom_components/easee/entity.py:195]: Is it finished?

Comment (astrandb) [reply custom_components/easee/entity.py]: Removed commented code.
No, translation only is done in frontend. The internal representation of state should not be affected. Can be verified in Developer Tools.

**Review by olalid — APPROVED** [review]

**Review by olalid — APPROVED** [review]

**Comment by astrandb** [comment]
Translation only is done in frontend. The internal representation of state should not be affected. Can be verified in Developer Tools.

**Pull request merged**